### DISCUSSION TITLE
Avances de estudio de los fundamentos de la estructura front-end/back-end del proyecto Armony

### DISCUSSION THREAD

**Discussion by yahirsaavedra** [createchsoftware/armony#3] [General]
¡Buenas tardes, compañeros!

Aquí se publican los avances de estudio. Consulten el comunicado enviado al grupo respectivo para más información.

Recuerden, cualquier duda me pueden preguntar en presencial, y por supuesto, también pueden ponerla aquí, Discord o Whatsapp. Muchas gracias.

@createchsoftware/programadores @createchsoftware/testers

**Comment by NICOLExALV** [comment]
Avances de estudio:
[Reporte AS1.pdf](https://github.com/createchsoftware/armony/files/14477097/Reporte.AS1.pdf)
_Nicole Alejandra Lugo Villa_

**Reply by NICOLExALV** [reply]
Avances de estudio:
[Reporte AS2.pdf](https://github.com/createchsoftware/armony/files/14554446/Reporte.AS2.pdf)
_Nicole Alejandra Lugo Villa_

**Reply by NICOLExALV** [reply]
Avances de estudio:
[Reporte AS3.pdf](https://github.com/createchsoftware/armony/files/14631584/Reporte.AS3.pdf)
_Nicole Alejandra Lugo Villa_

**Comment by JorgeInge** [comment]
avance
[Reporte.pdf](https://github.com/createchsoftware/armony/files/14504665/Reporte.pdf)
jorge francisco lopez hernandez

**Comment by Julian-Sandoval-x** [comment]
[Avance (26-03).pdf](https://github.com/createchsoftware/armony/files/14506654/Avance.26-03.pdf)
Avance de estudios (26 feb - 03 marzo)
Sandoval Godinez Julian David

**Comment by ItsJorgeRdz03** [comment]
Avance de estudios.
Jorge Alberto Rosales Rodriguez
[Avance 07-03-24 Jorge Rosales.pdf](https://github.com/createchsoftware/armony/files/14534553/Avance.07-03-24.Jorge.Rosales.pdf)

**Comment by KrisG012** [comment]
Primer reporte de avance de estudios.
**Gutiérrez Barajas Cristian**
_Archivo adjunto:_ 
[Reporte de Avance No.1.pdf](https://github.com/createchsoftware/armony/files/14548274/Reporte.de.Avance.No.1.pdf)

**Comment by adriandrs** [comment]
Primer Reporte de Avance de Estudios
**Adrian Fernando Santelis Mendoza**
[Avance.09-03.pdf](https://github.com/createchsoftware/armony/files/14548365/Avance.09-03.pdf)

**Comment by antonioEsB** [comment]
Avance de estudio #1
Antonio Esparza Bañales
[Reporte No.1.pdf](https://github.com/createchsoftware/armony/files/14553674/Reporte.No.1.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para exposición individual del Departamento de Testing.

**Comment by DkzBrandon** [comment]
Avance de estudio #1
Badillo Jimenez Brandon Arturo 
[Reporte 1.pdf](https://github.com/createchsoftware/armony/files/14554019/Reporte.1.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para exposición individual del Departamento de Testing.

**Comment by Batmand0** [comment]
[Documento sin título.pdf](https://github.com/createchsoftware/armony/files/14554033/Documento.sin.titulo.pdf)
Armendariz Magaña Armando

**Reply by yahirsaavedra** [reply]
Recibido. Para exposición individual del Departamento de Testing.

**Reply by Batmand0** [reply]
[Reporte2.pdf](https://github.com/createchsoftware/armony/files/14631636/Reporte2.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para presentación del Departamento de Testing.

**Reply by Batmand0** [reply]
[Reporte3.pdf](https://github.com/createchsoftware/armony/files/14736747/Reporte3.pdf)

**Comment by Julian-Sandoval-x** [comment]
[Avance(04-11).pdf](https://github.com/createchsoftware/armony/files/14565301/Avance.04-11.pdf)
Avance 04-03 / 11 - 03

**Comment by KarimSoto** [comment]
karim soto
[avance.pdf](https://github.com/createchsoftware/armony/files/14567292/avance.pdf)

**Comment by javierpartida02** [comment]
Javier Contreras Partida
[Reporte de Avances.pdf](https://github.com/createchsoftware/armony/files/14567509/Reporte.de.Avances.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para exposición individual del Departamento de Testing.

Todos en Testing entregaron.

**Comment by yahirsaavedra** [comment]
Ya no se reciben más entregas de esta semana.

Este hilo se reabrirá después de clase (12 de marzo, 09:00 PDT).

**Comment by yahirsaavedra** [comment]
Hilo reabierto. Se aceptan entregas.

**Comment by JorgeInge** [comment]
#2avance jorge francisco lopez hernandez
[2Reporte.pdf](https://github.com/createchsoftware/armony/files/14581788/2Reporte.pdf)

**Comment by Julian-Sandoval-x** [comment]
[Reporte semanal (10 - 17 marzo).pdf](https://github.com/createchsoftware/armony/files/14630410/Reporte.semanal.10.-.17.marzo.pdf)
Semana 10/03 - 17/03

**Comment by antonioEsB** [comment]
Reporte de estudio segunda semana
Antonio Esparza Bañales
[ReporteEstudio No.2.pdf](https://github.com/createchsoftware/armony/files/14630706/ReporteEstudio.No.2.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para presentación del Departamento de Testing.

**Comment by javierpartida02** [comment]
**Informe de estudio** 
Semana 2
_Javier Contreras Partida_
[Informe Semanal de Avances de Estudio.pdf](https://github.com/createchsoftware/armony/files/14631588/Informe.Semanal.de.Avances.de.Estudio.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para presentación del Departamento de Testing.

**Comment by DkzBrandon** [comment]
Avance de estudio #2
Badillo Jimenez Brandon Arturo
[Reporte.2.pdf](https://github.com/createchsoftware/armony/files/14631950/Reporte.2.pdf)

**Reply by yahirsaavedra** [reply]
Recibido. Para presentación del Departamento de Testing.

Todos en Testing entregaron.

**Comment by KrisG012** [comment]
Segundo y últumo reporte de avance de estudios.
**Gutiérrez Barajas Cristian**
_Archivo adjunto:_
[Reporte de Avance No.2.pdf](https://github.com/createchsoftware/armony/files/14681938/Reporte.de.Avance.No.2.pdf)

**Comment by KarimSoto** [comment]
[AE2.pdf](https://github.com/createchsoftware/armony/files/14696360/AE2.pdf)
avance de estudio Karim Soto

**Comment by JorgeInge** [comment]
#3 avance jorge francisco lopez hernandez
[3Reporte.pdf](https://github.com/createchsoftware/armony/files/14697158/3Reporte.pdf)

**Comment by ItsJorgeRdz03** [comment]
2do. Avance de estudios.
Jorge Alberto Rosales Rodriguez
[Avance 21-03-24-Jorge-Rosales.pdf](https://github.com/createchsoftware/armony/files/14706823/Avance.21-03-24-Jorge-Rosales.pdf)

**Comment by adriandrs** [comment]
Segundo Avance de Estudios
Adrian Fernando Santelis Mendoza
[Avance.21-03 [Adrian Santelis].pdf](https://github.com/createchsoftware/armony/files/14706888/Avance.21-03.Adrian.Santelis.pdf)

**Comment by yahirsaavedra** [comment]
Todos en Programación entregaron. Se cierra periodo de entrega para dicho departamento. Los de Programación ya no tienen que entregar reportes, pero hay que recordar que tanto el estudio como aprendizaje continuan durante todo el proyecto.

Este hilo seguirá abierto para Testing hasta el lunes a las 00:00 hrs.

**Comment by antonioEsB** [comment]
Reporte de no.3
Antonio Esparza Bañales
[ReporteNo_3.pdf](https://github.com/createchsoftware/armony/files/14734255/ReporteNo_3.pdf)

**Comment by DkzBrandon** [comment]
Avance de estudio #3
Badillo Jimenez Brandon Arturo
[Reporte3.pdf](https://github.com/createchsoftware/armony/files/14739633/Reporte3.pdf)

**Comment by yahirsaavedra** [comment]
Se cierra la actividad. Gracias.
